.github/workflows/windows.yml

```diff
@@ -205,6 +200,7 @@ jobs:
       OV_TOKENIZERS_INSTALL_DIR: ${{ github.workspace }}\b\ov_tk
       OPENVINO_TOKENIZERS_REPO: ${{ github.workspace }}\src\tk
       BUILD_DIR: ${{ github.workspace }}\b\tk
+      OV_ROOT: ${{ github.workspace }}\b\ov\${{ needs.openvino_download.outputs.ov_package_path }}
```

Comment:
am I right that package_path is actually a package_folder(s) ?

Reply:
Yes, technically it's a folder, or a path to the folder with OV setupvars relative to the root of unzipped artifacts archive

Reply:
ov_package_path was removed since artifacts were re-organized so that the folder with setupvars is always a root